cosmwasm/contracts/crosschain-registry/Cargo.toml

```diff
@@ -31,27 +34,27 @@ imported = []
 optimize = """docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.12.11
+  cosmwasm/rust-optimizer:0.16.0
```

Comment:
nit: We can probably swap these to just use `cosmwasm/optimizer:0.16.0` instead of `rust/workspace-optimizer` as in the [docs](https://github.com/CosmWasm/optimizer)